examples/basic/lines/line_missing_points.py

```diff
@@ -6,5 +6,8 @@
 
 # add a line renderer with NaN values
 p.line([1, 2, 3, nan, 4, 5], [6, 7, 2, 4, 4, 5], line_width=2)
+# don't use native None in a line renderer, because it will be drawn as 0
+p.line([1, 2, 3, None, 4, 5], [6, 7, 2, 4, 4, 5], line_width=2,
+       line_dash="dashed", color="red")
```

Comment:
I don't find the result comprehensible, because the lines are indistinguishable where they overlap:

<img width="434" alt="Screenshot 2024-10-28 at 09 44 48" src="https://github.com/user-attachments/assets/8588339f-fc95-4c4b-9f0f-a5bf0262bd7b">

If we want to do this, then the lines need to be made more distinguishable:

<img width="427" alt="Screenshot 2024-10-28 at 09 44 35" src="https://github.com/user-attachments/assets/882472ea-9aaa-40d3-85fd-2241b0f2303a">

That's with `line_width=8, alpha=0.5` for the first line. At this point I'd suggest adding "line with NaN" and "line with  None" legend items too